x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/edit_policy.tsx

```diff
@@ -243,13 +244,15 @@ export const EditPolicy: React.FunctionComponent<Props> = ({ history }) => {
               <HotPhase />
 
               <EuiSpacer />
-
               <WarmPhase />
 
               <EuiSpacer />
-
               <ColdPhase />
 
+              <EuiSpacer />
+              <FrozenPhase />
+
+              <EuiSpacer />
```

Comment:
Adding a spacer between frozen and delete phases makes the connecting line go beyond frozen phase:
<img width="594" alt="Screenshot 2021-03-04 at 17 46 03" src="https://user-images.githubusercontent.com/6585477/109999639-b7b66780-7d12-11eb-9a5a-3202223bad79.png">
I think this spacer needs to go into delete phase to fix it. There is an [issue](https://github.com/elastic/eui/issues/4492) open in EUI repo to address 'gutter size' between 'comment items' but it is not being worked on currently.

Reply:
Great catch! 👍

Reply:
I moved to the `DeletePhase` and it indeed fixed the issue.

<img width="1072" alt="Screenshot 2021-03-05 at 10 47 35" src="https://user-images.githubusercontent.com/2854616/110105436-49b88180-7da0-11eb-93a6-198b02b8cc11.png">

Reply:
Thanks for fixing this, @sebelga!